hexrd/wppf/WPPF.py

```diff
@@ -1831,14 +1825,19 @@ def chebyshevfit(self):
             self._background.append(Spectrum(x=tth, y=self.init_bkg(tth)))
 
     # cubic spline fit of background using custom points chosen from plot
-    def splinefit(self, x, y, tth):
+    def splinefit(self, points):
         """
         03/08/2021 SS adding tth as input. this is the
         list of points for which background is estimated
         """
+        self._background = []
+        x = points[:,0]
+        y = points[:,1]
         cs = CubicSpline(x, y)
-        bkg = cs(tth)
-        return Spectrum(x=tth, y=bkg)
+        for i, s in enumerate(self._spectrum_expt):
+            tth = self._tth_list[i]
```

Comment:
It was `tth = self._spectrum_expt[i]._x` before. Is `self._tth_list[i]` the same, or better?

Reply:
Line 1062 is 
`self._tth_list = [s._x for s in self._spectrum_expt]`

So they should be the same.